app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,19 @@
+module Api
+  module V1
+    class UsersController < Api::V1::ApiController
+      def update
+        current_user.update!(user_params)
```

Comment:
Usually, a good approach when updating a record it's returning `head :no_content`. You can [take a look at this](https://stackoverflow.com/questions/14716151/why-does-rails-want-to-return-head-no-content-for-json-put-requests) where this behavior is discussed briefly

Reply:
Excellent! To make sure, can I consider PUT and PATCH as the same or similar kind of request? In this terms at least

Reply:
Yes, actually in REST you can handle edit requests with both 👍